data/settings.js

```diff
@@ -11020,7 +11120,7 @@ and as such share the connections.`
 	},
 
 	ldap_debug_level: {
-		tags: [ 'ldap' ],
+		tags: [ 'ldap', 'sieve-storage-ldap' ],
```

Comment:
I don't think we need separate sieve-storage-ldap. For now it's happens to be a duplicate implementation of lib-ldap, but that'll hopefully change some day. Documentation shouldn't treat them as separate things.